src/lib/custom-deploy/custom-deploy.ts

```diff
@@ -13,47 +13,74 @@ import { validateDeployFileExt, validateFilename } from '../../lib/validations/c
 const DEPLOY_MAX_FILE_SIZE = 4 * GB_IN_BYTES;
 const WPVIP_DEPLOY_TOKEN = process.env.WPVIP_DEPLOY_TOKEN;
 
+type CustomDeployInfo = {
+	success: boolean;
+	appId: number;
+	envId: number;
+	envType: string;
+	primaryDomainName: string;
+	launched: boolean;
+	customDeploysEnabled: boolean;
+};
+
+type ValidateMutationPayload = {
+	data?: {
+		validateCustomDeployAccess: CustomDeployInfo;
+	} | null;
+};
+
 export function isSupportedApp( app: App ): boolean {
```

Comment:
We can remove this now I think since it looks like we're only doing the validation on the Parker end?